.python-version

```diff
@@ -1,7 +1,6 @@
-2.6.9
 2.7
-3.2
-3.3.6
 3.4.3
+3.7.6
+3.8.2
 pypy-2.5.1
 pypy3-2.4.0
```

Comment:
With Python 2.7 having reached its end of life, I think it's reasonable to remove support. Also, take care to update the `README` and release notes.